src/persist-client/src/operators/shard_source.rs

```diff
@@ -66,11 +66,10 @@ use crate::{Diagnostics, PersistLocation, ShardId};
 /// using [`timely::dataflow::operators::generic::operator::empty`].
 ///
 /// [advanced by]: differential_dataflow::lattice::Lattice::advance_by
-pub fn shard_source<'g, K, V, T, D, F, DT, G>(
+pub fn shard_source<'g, K, V, T, D, F, DT, G, C>(
```

Comment:
this is one of the longest list of generics ive ever seen haha

Reply:
Yeah... a bit sad but seems hard to avoid in this case!